src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+
+### Use Cases of RBD Live Migration
+
+* Disaster Recovery and Data Migration\
+_Scenario_: An organization runs mission-critical applications on a primary Ceph cluster
+in one data center. Due to an impending maintenance window, potential hardware
+failure, or a disaster event, they need to migrate RBD images to a secondary
+Ceph cluster in a different location.\
+_Benefit_: Live migration ensures that applications using RBD volumes can continue
+functioning with minimal downtime and no data loss during the transition to the
+secondary cluster.
+
+* Bursting and Workload Distribution\
+_Scenario_: An organization operates a Ceph cluster that accommodates routine workloads but
+occasionally requires extra capacity during peak usage. By migrating RBD images
+to an external Ceph cluster (possibly deployed in a cloud) they can temporarily
+scale operations then scale then back.\
+_Benefit_: Dynamic workload balancing helps admins leverage external resources
+only when needed, reducing operational costs and improving scalability.
+
+* Data Center Migration\
+_Scenario_: An organization is migrating their infrastructure from one physical data
+center to another due to an upgrade, consolidation, or relocation. All RBD images
+from the source Ceph cluster need to be moved to a destination Ceph cluster in the
+new location.\
+_Benefit_: Live migration minimizes disruptions to services during data center
+migrations, maintaining application availability.
+
+* Compliance and Data Sovereignty\
+_Scenario_: A organization must comply with local data residency regulations that
+require sensitive data to be stored within specific geographic boundaries.
+Data held in RBD images thus must be migrated from a general-purpose Ceph
+cluster to one dedicated to and within the regulated region.\
+_Benefit_: The live migration feature enables seamless relocation of RBD data
+without halting ongoing operations, ensuring compliance with regulations.
+
+* Multi-Cluster Load and Capacity Balancing\
+_Scenario_: An organization runs multiple Ceph clusters to handle high traffic
+workloads. To prevent overloading any single cluster, they redistribute
+RBD images among clusters as workload patterns shift.\
+_Benefit_: Live migration allows for efficient rebalancing of workloads across
+Ceph clusters, optimizing resource utilization and performance.
+
+* Dev/Test to Production Migration\
+_Scenario_: Developers run test environments on a dedicated Ceph cluster.
+After testing is complete, production-ready RBD images can be migrated to the
+production Ceph cluster without data duplication or downtime.\
+_Benefit_: Simplifies the process of promoting test data to production
+while maintaining data integrity.
+
+* Hardware Lifecycle Management\
+_Scenario_: A Ceph cluster is running on older hardware nearing the end of
```

Comment:
s/hardware/hardware that is/

This isn't absolutely necessary, but I like relative clauses. If you're tired of making alterations, don't feel that this is absolutely necessary.